app/js/main.js

```diff
@@ -204,15 +204,6 @@ function startup() {
   });
 
   /* ======= Listeners ======= */
-  // Change resolution
-  document.querySelector("#capture-options li:first-child").addEventListener("click", function() {
-    notification.info("This feature is not yet available!");
-  });
-
-  // Change camera
-  document.querySelector("#capture-options li:last-child").addEventListener("click", function() {
-    notification.info("This feature is not yet available!");
-  });
```

Comment:
These were removed because the `Change resolution` and `Change camera` options in the sidebar that these listeners refer to were also removed. These in turn were removed because firstly they are just placeholders and secondly because the partially complete WebRTC work for these options will use elements in the UI located elsewhere.